restock.lic

```diff
@@ -37,10 +36,8 @@ class Restock
       item['buy_num'] = buy_num
       items_to_restock.push(item)
     end
-    town = @settings.hometown
-    town = items_to_restock.first['hometown'] if items_to_restock.first.key?('hometown')
-    coin_string = coin_needed.to_s + ' copper ' + town_currency(town)
-    get_money_from_specific_bank(coin_string, town) if coin_needed > 0
+    money_string = coin_needed.to_s + ' copper ' + town_currency(town)
```

Comment:
💡 This is a case where I prefer string interpolation, something like `"#{coin_needed.to_s} copper #{town_currency(town)}"`

Reply:
Just so used to stringstream, lol. I gotcha.